sdk/keyvault/TEST_RESOURCES_README.md

```diff
@@ -0,0 +1,130 @@
+will be pre-filled in a drop-down menu. You can either pick one resource group from this menu,
+or click on the "Create new" link right under the drop-down menu to create a new resource group.
+You can read more about Resource groups [here][ResourceGroups].
+
+**This field is required.**
+
+### The "Location" field.
+
+Azure has more global regions than any other cloud provider.
+Once you've picked a subscription and a resource group,
+you'll be able to pick a location. 
+You can read more about Azure locations [here][AzureLocations].
+
+**This field is required.**
+
+### The "Base Name" field.
+
+Under "Settings", the "Base Name" will serve us as a prefix
+to determine the name of the resources that we will create,
+so that you can easily find them later.
+
+This field gets automatically generated for us through our
+CI pipelines. It is **required** to replace this default value
+with any combination of alphanumeric characters that you might
+prefer. Keep in mind that if you have previously generated
+resources with the same alphanumeric combination that you're
+providing, Azure will likely fail, requiring you to insert
+a different value.
+
+### The "Tenant Id" field.
+
+To fill the **Tenant Id** field, you may copy it from the one available on the
+[overview page of the Azure Active Directory section of the Azure portal][AADOverview],
+or obtain it from the overview page of a recently created Azure Application.
+Here's [how to use the portal to create an Azure AD application and service principal that can access resources][HowToAADApp].
+
+**This field is required.**
+
+### The "Test Application Oid" field.
+
+To fill the **Test Application Oid** field, you may get it from the overview page of a recently created Azure application.
+
+If you don't have an Azure AD application, follow: [How to use the portal to create an Azure AD application and service principal that can access resources][HowToAADApp].
+Once created, go to your application's overview page, and you'll see an "Object ID" property. This is the one that you want to copy and paste on the form.
+
+**This field is required.**
+
+### Terms and conditions
+
+Make sure to read the Terms and Conditions before purchasing these resources.
+
+### Verifying your deploy
+
+Once your purchase is completed, a notification badge will appear on the top right of the portal,
+indicating that the resource is being created. You can go to your resource group's page,
+then go to the **Deployments** section to see a history of all the deployments you've made.
+
+The deployments that have used an ARM template will say in their names **Microsoft.Template**
+If you click in one of them, you'll see at the left "Outputs" as one of the menu options.
+If your deployment has succeeded, the outputs will contain the name of the KeyVault that was just created.
+
+## Running the KeyVault client tests
+
+Once you have the resources ready to run our integration tests,
```

Comment:
Once you have the resources ready to run our integration tests,
continue reading [how to run our integration tests](./test/README.md).

(maybe)

After your resources have been deployed you can read about [how to run our integration tests](./test/README.md).

Reply:
I'll take it!